Packs/FireEyeHX/Integrations/FireEyeHXEventCollector/FireEyeHXEventCollector.py

```diff
@@ -132,22 +130,19 @@ def get_events_command(
 
 
 def fetch_events(
-    client: Client, max_fetch: str, first_fetch: str, resolution: str = None, min_id: str = None,
-        should_push_events: bool = True
+    client: Client, max_fetch: str, first_fetch: str, min_id: str = None, should_push_events: bool = True
 ) -> list:
     """
     Fetches events from fireeye.
     """
 
-    # incase we want to retrieve all the resolution type, no need for this parameter.
-    if resolution and resolution.lower() == 'all':
-        resolution = None
+    filter_query_str = None
+    if not min_id:
+        to_date = (datetime.now() + timedelta(days=1)).strftime(DATE_FORMAT)
```

Comment:
why to add another day? and not just now?

Reply:
i prefer to add 1 day buffer

Reply:
I changed it to :
`to_date = datetime.now().strftime(DATE_FORMAT)`